file_upload.sh

```diff
@@ -0,0 +1,2 @@
+az storage container create -n "container-1" --public-access blob
```

Comment:
It would be very easy to do so if the startup/tear down were controlled by scripts, however the intention was to keep it simple and control the containers only with docker-compose up/down commands.
For production / anything outside of local development, this script / component is useless. The container name is declared in the docker-compose.yml and can be set with an env. variable, but will default to container-1 for local dev.